trivy-operator.yaml

```diff
@@ -0,0 +1,77 @@
+package:
+  name: trivy-operator
+  version: "0.25.0"
+  epoch: 1
+  description: "Kubernetes-native security toolkit that finds and reports vulnerabilities and misconfigurations"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates
+      - trivy
+      - tzdata
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aquasecurity/trivy-operator
+      tag: v${{package.version}}
+      expected-commit: 0816542808e81dc318a34a1ce8a5cd004c59dfdd
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/containerd/containerd@v1.7.27
+        github.com/containerd/containerd/v2@v2.0.4
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        golang.org/x/oauth2@v0.27.0
+        helm.sh/helm/v3@v3.17.3
+
+  - uses: go/build
+    with:
+      packages: "./cmd/trivy-operator"
+      output: trivy-operator
+      ldflags: |-
+        -X github.com/aquasecurity/trivy-operator/pkg/version.Version=${{package.version}}
+        -X github.com/aquasecurity/trivy-operator/pkg/version.Commit=melange-build
+        -X github.com/aquasecurity/trivy-operator/pkg/version.BuildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+update:
+  enabled: true
+  github:
+    identifier: aquasecurity/trivy-operator
+    strip-prefix: v
+
+test:
+  environment:
+    environment:
+      OPERATOR_NAMESPACE: "trivy-system"
+      KUBERNETES_SERVICE_HOST: "127.0.0.1"
+      KUBERNETES_SERVICE_PORT: "32764"
+    contents:
+      packages:
+        - ${{package.name}}
+  pipeline:
+    - name: "Basic binary verification"
+      runs: |
+        if [ ! -x /usr/bin/trivy-operator ]; then
+          echo "Error: trivy-operator binary not found or not executable"
+          exit 1
+        fi
+
+        if ! /usr/bin/trivy-operator version 2>&1 | grep -q "Starting operator"; then
+          echo "Error: Binary doesn't produce expected startup message"
+          exit 1
+        fi
+
+        echo "Basic binary verification passed"
```

Comment:
we can simplify this by doing `/usr/bin/trivy-operator version 2>&1 | grep -q "Starting operator`